app/views/admin/products_v3/_filters.html.haml

```diff
@@ -17,6 +17,9 @@
     = select_tag :category_id, options_for_select(category_options, category_id),
       include_blank: t('.all_categories'), class: "fullwidth",
       data: { "controller": "tom-select", 'tom-select-placeholder-value': t('.search_for_categories')}
+  .tags
+    = label_tag :tags_name_in, t('.tags.label')
+    = select_tag :tags_name_in, options_for_select(ActsAsTaggableOn::Tag.for_context("variant_tag").distinct.order(:name).pluck(:name), @tags), { class: "fullwidth", multiple: true }
```

Comment:
Nice, but I think it would be better to have the db query in the controller, and pass it through to the template.